contents/docs/product-analytics/troubleshooting.mdx

```diff
@@ -250,7 +250,9 @@ A smaller time period means less data to process. You can always adjust the peri
 
 #### 3. Change the person properties mode
 
-PostHog provides multiple modes for the behavior of person property filters. You can find and change them in [your project settings](https://us.posthog.com/settings/project#persons-on-events). 
+PostHog provides multiple modes for the behavior of person property filters. You can find and change them in [your project settings](https://us.posthog.com/settings/project#persons-on-events).
```

Comment:
Let's remove this link, and the screenshot, as we don't want new customers to ask for access. This section is about how to speed up queries, so we can just say "avoid using pdi.person.properties, just use person.properties instead".